src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,99 @@
+package Duke;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.time.format.DateTimeFormatter;
+
+public class Storage {
+
+    private String filePath;
+
+    /**
+     * Creates Storage instance.
+     * @param filepath the file name.
+     */
+    public Storage(String filepath) {
+        this.filePath = filepath;
+    }
+
+    /**
+     * Saves the tasklist.
+     * @param items the stringified tasklist.
+     */
```

Comment:
Might it be better to have an empty line between the description and parameter section?